manual/sphinx/user_docs/advanced_install.rst

```diff
@@ -258,12 +258,27 @@ appropriately.
 OpenMP
 ------
 
-BOUT++ can make use of Single-Instruction Multiple-Data (SIMD)
-parallelism through OpenMP. To enable OpenMP, use the
+BOUT++ can make use of OpenMP parallelism. To enable OpenMP, use the
 ``--enable-openmp`` flag to configure::
 
     ./configure --enable-openmp
 
+OpenMP can be used to parallelise in more directions than can be
+achieved with MPI alone. For example, it is currently difficult to
+parallelise over the X, Z plane using pure MPI if FCI is used.
```

Comment:
It is currently impossible to parallelise over Z using pure MPI for any choice of parallel transform!